.github/workflows/bot-pr-comment.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         fetch-depth: 0
     - name: Add appropriate labels
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
```

Comment:
What's the difference between `secrets.GITHUB_TOKEN` and `secrets.BOT_TOKEN`?

Reply:
GITHUB_TOKEN is an automatically generated and temporary token for **github-actions[bot]**, which has the annoying property that pushes from this token don't trigger subsequent Github Actions (like Windows/macOS CI jobs) to avoid infinite loops in Github Actions by default.
BOT_TOKEN is a manually generated personal authentication token for @ginkgo-bot which circumvents this limitation.
So in the future, comments will come from **ginkgo-bot** instead of **github-actions[bot]** and the CI jobs will run after every push